sql/install.sql

```diff
@@ -1,367 +1,367 @@
+(50140, 50000, "Pièces de rechange et accessoires (auto)"),
+(50150, 50000, "Pièces de rechange et accessoires (autres)"),
+(50170, 50000, "Montres, horlogerie (hors bijoux)"),
+(50330, 50000, "Articles de camping, de pêche"),
+(50350, 50000, "Articles de sport (hors vêtement)"),
+(50360, 50000, "Instruments de musique et accessoires"),
+(50380, 50000, "Matériel de chauffage, chaudronnerie"),
+(50390, 50000, "Matériel de labo, optique, de mesure"),
+(50395, 50000, "Matériel électrique, transfo., câbles"),
+(50400, 50000, "Fournitures de bureau, papeterie, recharges"),
+(50420, 50000, "Moteurs"),
+(50430, 50000, "Motos, scooters"),
+(50440, 50000, "Vélos, cycles sans moteur"),
+(50450, 50000, "Outillage, outils, bricolage"),
+(50490, 50000, "Plomberie, tubes plastiques"),
+(50500, 50000, "Quincaillerie, robinetterie, serrurerie"),
+(60000, 0, "Mobilier et décoration"),
+(60100, 60000, "Mobilier d\'habitation"),
+(60102, 60000, "Mobilier de bureau"),
+(60105, 60000, "Mobilier démonté sous emballage"),
+(60108, 60000, "Mobilier ancien (antiquité)"),
+(60110, 60000, "Electroménager "),
+(60112, 60000, "Petit électroménager, petits appareils ménagers"),
+(60120, 60000, "Objets ou tableaux cotés, de collection, miroirs, vitres"),
+(60122, 60000, "Objets et tableaux courants "),
+(60124, 60000, "Lampes, luminaire"),
+(60126, 60000, "Tapis"),
+(60128, 60000, "Toiles, rideaux, draps"),
+(60129, 60000, "Sanitaires, verres, cristallerie, bibelots"),
+(60130, 60000, "Autres objets fragiles et sculptures"),
+(70000, 0, "Effets personnels, cadeaux"),
+(50180, 70000, "Cadeaux, cadeaux entreprise"),
+(70100, 70000, "Bagages, valises, malles"),
+(70200, 70000, "Petit déménagement, cartons, effets personnels");
+-- REQUEST --
+DROP TABLE IF EXISTS `{PREFIXE}emc_dimensions`;
+-- REQUEST --
+CREATE TABLE IF NOT EXISTS `{PREFIXE}emc_dimensions` (
+	`id_ed` int(3) NOT NULL AUTO_INCREMENT,
+	`length_ed` int(3) NOT NULL,
+	`width_ed` int(3) NOT NULL,
+	`height_ed` int(3) NOT NULL,
+	`weight_from_ed` float NOT NULL,
+	`weight_ed` float NOT NULL,
+	PRIMARY KEY (`id_ed`)
+)  DEFAULT CHARSET=utf8;
+-- REQUEST --
+INSERT INTO {PREFIXE}emc_dimensions (`id_ed`, `length_ed`, `width_ed`, `height_ed`, `weight_from_ed`, `weight_ed`) VALUES
+(1, 18, 18, 18, 0, 1),
+(2, 22, 22, 22, 1, 2),
+(3, 26, 26, 26, 2, 3),
+(4, 28, 28, 28, 3, 4),
+(5, 31, 31, 31, 4, 5),
+(6, 33, 33, 33, 5, 6),
+(7, 34, 34, 34, 6, 7),
+(8, 36, 36, 36, 7, 8),
+(9, 37, 37, 37, 8, 9),
+(10, 39, 39, 39, 9, 10),
+(11, 44, 44, 44, 10, 15),
+(12, 56, 56, 56, 15, 20),
+(13, 57, 57, 57, 20, 50);
+-- REQUEST --
+DROP TABLE IF EXISTS `{PREFIXE}emc_documents`;
+-- REQUEST --
+CREATE TABLE IF NOT EXISTS `{PREFIXE}emc_documents` (
+	`id_ed` int(11) NOT NULL AUTO_INCREMENT,
+	`{PREFIXE}orders_id_order` int(10) unsigned NOT NULL,
+	`{PREFIXE}cart_id_cart` int(10) unsigned NOT NULL,
+	`link_ed` varchar(255) NOT NULL,
+	`generated_ed` int(1) NOT NULL DEFAULT 0,
+	`type_ed` varchar(100) NOT NULL,

```

Comment:
Because the database structure has been modified, you have to create an upgrade file which will alter your table structure during the upgrade.